conf/layer.conf

```diff
@@ -36,5 +36,5 @@ BBFILES_DYNAMIC += " \
     multimedia-layer:${LAYERDIR}/dynamic-layers/multimedia-layer/*/*/*.bbappend \
 "
 
-DEFAULT_TEST_SUITES_remove_rpi = "parselogs"
-DEFAULT_TEST_SUITES_append_rpi = " parselogs_rpi"
+DEFAULT_TEST_SUITES:remove:rpi = "parselogs"
+DEFAULT_TEST_SUITES:append:rpi = " parselogs:rpi"
```

Comment:
How does this one work?

Reply:
I think the testcase is called "parselogs_rpi".

Reply:
will fix